### PR TITLE
feat: add anonymous contact option to contact form

### PR DESCRIPTION
**Pull request by itislu**
### Issues still to solve:
- [x] Submitting a message with the anonymous option doesn't work.
- [x] Refreshing the page causes an error (introduced by saving email field in localStorage in ba0843b)
- [x] The user's typed in email does not get restored from localStorage.

### CODE REVIEW

File: src/app/routes/contact.tsx [hunk 1]
```diff
@@ -56,8 +56,8 @@ export async function action({ request }: ActionFunctionArgs) {
                 embed.author = {
                     name: session.login,
                     url: `https://profile.intra.42.fr/users/${session.login}`,
-                    icon_url: session.imageUrl,
```
Comment (itislu) [inline src/app/routes/contact.tsx]: Why remove the icon?

Comment (ifaoji) [reply src/app/routes/contact.tsx]: Currently it is not stored in the session anymore

Comment (itislu) [reply src/app/routes/contact.tsx]: Oh so we didn't get icons anymore anyway?

File: src/app/routes/contact.tsx [hunk 2]
```diff
@@ -129,11 +129,16 @@ export default function Contact() {
...
-        const savedContactOption = localStorage.getItem('contact-option');
         if (savedMessage) setMessage(savedMessage);
+
+        const savedAnonymousOption = localStorage.getItem('anonymous-option');
         if (savedAnonymousOption) setAnonymousOption(savedAnonymousOption);
+
+        const savedContactOption = localStorage.getItem('contact-option');
         if (savedContactOption) setContactOption(savedContactOption);
+
+        let savedEmail = localStorage.getItem('contact-email');
```
Comment (itislu) [inline src/app/routes/contact.tsx:140]: Btw, is there a reason why all the others are `const` but this one not?

Comment (ifaoji) [reply src/app/routes/contact.tsx:140]: I changed it to let for some debugging purposes, and then forgot to change it back to const :facepalm:

**Review by ifaoji — APPROVED** [review]

**Pull request merged**